src/test/packages/36-tarball-image-references/zarf.yaml

```diff
@@ -0,0 +1,24 @@
+kind: ZarfPackageConfig
+metadata:
+  name: tarball-image-reference
+  description: test if we can create a deploy a package which uses an image from a tarball
+components:
+  - name: baseline
+    required: true
+    manifests:
+      - name: multi-games
+        namespace: tarball-dos-games
+        files:
+          - manifests/deployment.yaml
+          - manifests/service.yaml
+    images:
+      - zarf-game.tar
```

Comment:
I believe we will not want to commit this tar file to git history. One way to get around this is to use the `exec.CmdWithPrint` function within the test to run a docker command to save the image. Make sure the test also cleans up the image. 

@Racer159 Can you validate that this would be a good approach. Any other ideas ?

Reply:
Correct (it isn't in git history here because we `.gitignore` all tar files) - the best way would actually likely be a `-cmd: ./zarf tools registry pull defenseunicorns/zarf-game:multi-tile-dark zarf-game.tar` in an onCreate before action: https://docs.zarf.dev/docs/create-a-zarf-package/component-actions#additional-action-use-cases that way the package works on it's own outside of tests too.

Reply:
```yaml
    actions:
      onCreate:
        before:
          - cmd: ./zarf tools registry pull defenseunicorns/zarf-game:multi-tile-dark zarf-game.tar
```